vega_sim/scenario/curve_market_maker/scenario.py

```diff
@@ -263,6 +263,7 @@ def set_up_background_market(
     def run_iteration(
         self,
         vega: VegaServiceNull,
+        network: str,
```

Comment:
Can we make these Optional everywhere too please? Given that I think it's totally unused in the default Nullchain case?

Reply:
Sure thing, I will make it optional in any scenario where it is unused, but will keep as needed for scenarios which can run either the `VegaServiceNullchain` service or `VegaServiceNetwork`